plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryPager.php

```diff
@@ -49,123 +49,29 @@ protected function get_block_type_uses_context() {
 	 * @return string Rendered block type output.
```

Comment:
We dont return a `string` if `$product` is falsey. Perhaps we should return an empty string